### PR TITLE
Add ability to disable dataviews

### PR DESCRIPTION
**Pull request by geoffreymuchai**
Disable specific dataviews by updating the disabled views list

### CODE REVIEW

File: src/hatti/utils.cljs [hunk 1]
```diff
@@ -54,3 +54,8 @@
   "Create an appropriately pluralized string prefix by number."
   [number kind]
   (join " " [number (if (= 1 number) kind (plural kind))]))
+
+(defn in?
```
Comment (pld) [inline src/hatti/utils.cljs]: Cool, at some point in the future we need a clojure utils library, we have this in zebra, hatti, and milia

File: src/hatti/views/dataview.cljs [hunk 2]
```diff
@@ -100,8 +100,9 @@
             dataviews (->> app-state :views :all
                            (map dataview-map) (remove nil?))
             dv->link (fn [{:keys [view label]}]
-                       (if (and (= view :map) no-geodata?)
-                         [:a {:class "inactive" :title "No geodata"}
+                       (if (or (and (= view :map) no-geodata?)
+                               (in? (-> app-state :views :disabled) view))
+                         [:a {:class "inactive" :title "No data"}
```
Comment (pld) [inline src/hatti/views/dataview.cljs]: If it's disabled I don't think we should day "no data"

**Comment by pld** [comment]
Can we add a test for the disabled functionality to here, https://github.com/onaio/hatti/blob/master/test/hatti/views/dataview_test.cljs

File: src/hatti/views/dataview.cljs (hunk 2 above)
Comment (geoffreymuchai) [inline src/hatti/views/dataview.cljs]: We can just say disabled for now

File: src/hatti/utils.cljs (hunk 1 above)
Comment (geoffreymuchai) [inline src/hatti/utils.cljs]: This sounds good to me

File: src/hatti/views/dataview.cljs (hunk 2 above)
Comment (pld) [inline src/hatti/views/dataview.cljs]: That's OK, it's a separate case from the "No geodata" and should be handled
as such.

Eventually we'll want to translate this, but we can ignore that for now.

**Comment by pld** [comment]
I can't figure out why this isn't failing. The test failure seem unrelated to the code this PR changes. Could use another set of eyes on this...

**Comment by geoffreymuchai** [comment]
@pld Tests have been added

**Comment by pld** [comment]
Yeah, I noticed, but it's not passing

**Comment by royrutto** [comment]
Is this still required? Can it be closed without merging?

**Comment by pld** [comment]
this slipped through the cracks, I think we should merge it, I can take on the rebase since it relates to some other code I've been working on

**Comment by pld** [comment]
@royrutto ok, rebased and passing tests

**Pull request merged**